src/main/java/mate/project/model/Book.java

```diff
@@ -8,9 +8,13 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Data
```

Comment:
Omit using `@Data` annotation in the entities. It may lead to unexpected problems later on. Better to use:
```java
@Getter
@Setter
@NoArgsConstructor
```